go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.9.0
-	go.temporal.io/api v1.32.0
+	go.temporal.io/api v1.32.1-0.20240423190756-d1ff87aac800
```

Comment:
We are not planning on merging this to `master` until a tagged version of API is done (we are letting other in-flight API projects drive the need for tagging)

Reply:
This has been updated